### PR TITLE
Fix fee levels

### PR DESCRIPTION
**Pull request by jjeffryes**
This will fix the fee levels to always use PRIORITY, NORMAL, and ECONOMIC, instead of low, medium and high.

It will also correct the advanced tab to show errors with the fee levels on save, and homogenizes the language used so it always matches the server to avoid confusion.

### CODE REVIEW

File: js/languages/en-US.json [hunk 1]
```diff
@@ -1363,14 +1362,18 @@
     "provideReview": "Please provide a review.",
     "provideRating": "Please select a rating."
   },
+  "localStorageModelErrors": {
```
Comment (rmisio) [inline js/languages/en-US.json]: The model name is localSettings. Can the translation key be consistent with that?

Comment (jjeffryes) [inline js/languages/en-US.json]: Done.

File: js/models/wallet/Spend.js [hunk 2]
```diff
@@ -117,7 +117,8 @@ export default Spend;
 export function spend(fields) {
   const attrs = {
     currency: app && app.settings && app.settings.get('localCurrency') || 'BTC',
-    feeLevel: app && app.localSettings && app.localSettings.get('defaultTransactionFee') || 'high',
+    feeLevel: app &&
+    app.localSettings && app.localSettings.get('defaultTransactionFee') || 'PRIORITY',
```
Comment (rmisio) [inline js/models/wallet/Spend.js]: @jjeffryes Could you ensure this PR passes lint. If so, go ahead and merge it.

Comment (jjeffryes) [inline js/models/wallet/Spend.js]: Will do.

**Pull request merged**